### PR TITLE
Expand implementation-defined limits on func/block param/result counts

### PR DESCRIPTION
**Pull request by wingo**

### CODE REVIEW

**Review by rossberg — APPROVED** [review]

**Comment by Ms2ger** [comment]
@rossberg is this blocked on anything?

**Comment by rossberg** [comment]
Ah, sorry, no, I didn't realise you don't have commit rights. Done.

**Pull request merged**
